README.md

```diff
@@ -129,6 +129,7 @@ may lead to messages being silently dropped.
 * `persistence` - Defaults to false (messages will not survive a broker restart)
 * `max_keepalive` - defaults to 65535 and, from version 2.0.12, `SetKeepAlive(0)` will result in a rejected connection 
 by default.
+* `token.Wait()` - Will return successfully before a message has been delivered if the client needs to reconnect to the broker before the message has been delivered.
```

Comment:
Move up above the "If using Mosquitto then there are a range of fairly common issues:" line (as this is not specific to Mosquitto). I would suggest something like the following (to mirror the format of previous comments and reflect the impact of options):

Publish tokens will complete when the connection is lost (in most cases `token.Error()` will return `nil` - see the `internalConnLost` function in `client.go` for full details). The message may be delivered to the broker when the connection is re-established (either via the `AutoReconnect` functionality or manually depending upon the `CleanSession`/`Store` `ClientOptions`)